<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1476</width>
    <height>1001</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>1</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="4" column="0">
          <widget class="QComboBox" name="camera_select"/>
         </item>
         <item row="4" column="1">
          <widget class="QPushButton" name="remove_cam">
           <property name="text">
            <string>Remove camera</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QComboBox" name="model_select"/>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="remove_fig">
           <property name="text">
            <string>Remove figure</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0" colspan="2">
          <widget class="QPushButton" name="pb_load_camera">
           <property name="text">
            <string>Load camera</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="pb_load_model">
           <property name="text">
            <string>Load model</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_6">
         <item row="1" column="0" colspan="3">
          <widget class="QPushButton" name="shift_fig">
           <property name="text">
            <string>Shift figure</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="dsb_model_shift_y"/>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="dsb_model_shift_z"/>
         </item>
         <item row="0" column="0">
          <widget class="QDoubleSpinBox" name="dsb_model_shift_x"/>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_4">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="dsb_model_rot_y"/>
         </item>
         <item row="0" column="0">
          <widget class="QDoubleSpinBox" name="dsb_model_rot_x"/>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="dsb_model_rot_z"/>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QPushButton" name="rotate_fig">
           <property name="text">
            <string>Rotate figure</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_5">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="dsb_model_scale_y"/>
         </item>
         <item row="0" column="0">
          <widget class="QDoubleSpinBox" name="dsb_model_scale_x"/>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="dsb_model_scale_z"/>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QPushButton" name="scale_fig">
           <property name="text">
            <string>Scale figure</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_7">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QDoubleSpinBox" name="dsb_camera_shift_x"/>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="dsb_camera_shift_y"/>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="dsb_camera_shift_z"/>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QPushButton" name="shift_cam">
           <property name="text">
            <string>Shift camera</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_6">
        <property name="title">
         <string/>
        </property>
        <layout class="QGridLayout" name="gridLayout_7">
         <item row="0" column="0">
          <widget class="QDoubleSpinBox" name="dsb_camera_rot_x"/>
         </item>
         <item row="0" column="1">
          <widget class="QDoubleSpinBox" name="dsb_camera_rot_y"/>
         </item>
         <item row="0" column="2">
          <widget class="QDoubleSpinBox" name="dsb_camera_rot_z"/>
         </item>
         <item row="1" column="0" colspan="3">
          <widget class="QPushButton" name="rotate_cam">
           <property name="text">
            <string>Rotate camera</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1476</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
